src/components/atoms/Button/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './style.css';
+
+const Button = ({ className, arrowDirection, onClick }) => {
+    return (
+        <button className={className} onClick={onClick}>
```

Comment:
why to have onclick if you can place onclick on `Button` component

Reply:
I planed use this onClick on carousel buttons. If i don't use it i will delete it later

Reply:
Strange implementation, should be  `onClick={() => onClick()}`  or remove it.

Reply:
Removed it.